go/chat/helper.go

```diff
@@ -18,168 +18,71 @@ import (
 	"github.com/keybase/client/go/protocol/keybase1"
 )
 
-func SendTextByName(ctx context.Context, g *globals.Context, name string, topicName string,
```

Comment:
Can we keep these functions? Its nice to be able to just send a message without having to create an object.